guides/install.md

```diff
@@ -18,12 +18,31 @@ The process of installing SideStore onto your device requires the **one time use
 
 "Burner" Apple IDs are **highly** recommended when using anisette servers created by others. See more about [creating your own anisette server.](/guides/custom-anisette)
 
-You must enable "Developer Mode" in your iOS security settings. Additionally, you must *Trust* your personal developer certificate after installing with your Apple ID.
+You must enable "Developer Mode" in your iOS security settings on iOS 16+. Additionally, you must *Trust* your personal developer certificate after installing with your Apple ID.
+
+**You will need the wireguard vpn app and use the wireguard config provided and share it to the wireguard app.**
+
+[Wireguard](https://apps.apple.com/us/app/wireguard/id1441195209)
+
+[Wireguard Config](https://github.com/SideStore/SideStore/releases/download/0.1.1/SideStore.conf)
```

Comment:
I feel like we should host this file in the assets repo or something instead of linking to v0.1.1 release everywhere, also because wasn't there a new revision with 1.1.1.1 dns or whatever?

Reply:
The newest one removed 1.1.1.1 because of connection issues. I only replaced the config that was in 0.1.1 so no it's latest rn. We haven't had a new version since.

Reply:
Latest changes should help with doing new changes without touching the docs.